doc(setting): eventual danger zone setting update (#858)
longhorn/website: content/docs/1.6.0/advanced-resources/deploy/priority-class.md

# Setting Priority Class

> Longhorn system contains user deployed components (e.g, Longhorn manager, Longhorn driver, Longhorn UI) and system managed components (e.g, Instance manager, Engine image, CSI driver, etc.)

Reviewer: ```suggestion
Longhorn consists of user-deployed components (for example, Longhorn Manager, Longhorn Driver, and Longhorn UI) and system-managed components (for example, Instance Manager, CSI Driver, and engine images). 
```

Author: Updated. Thanks

# Usage

> Suggest detaching all Longhorn volumes before modifying Priority Class settings.

Reviewer: ```suggestion
For best results, detach all Longhorn volumes before modifying Priority Class settings.
```

Author: Modified. Thanks

> For best results, detach all Longhorn volumes before modifying Priority Class settings.

Reviewer 2: Don't use `best results`

Author: Modified and all others. Thanks

> To apply a setting immediately, stop all workloads and detach all Longhorn volumes before modifying Priority Class settings.

Reviewer: ```suggestion
To ensure that your preferred Priority Class settings are immediately applied, stop all workloads and detach all Longhorn volumes before configuring the settings.
```

Longhorn temporarily becomes unavailable when all components are restarted.
Don't operate the Longhorn system after modifying the Priority Class setting, as the Longhorn components will be restarting.

> For Longhorn volumes all detached, the customized setting will be applied to system-managed components immediately.
> For one or more volumes in use, the customized setting will be applied to the Instance Manager without engine and replica instances running, and you are required to reconfigure this setting when all volumes are finally detached.

Reviewer: ```suggestion
When all Longhorn volumes are detached, the customized settings are immediately applied to the system-managed components.
When one or more Longhorn volumes are still attached, the customized settings are applied to the Instance Manager only when no engines and replica instances are running. You might be required to reconfigure the settings after detaching the remaining volumes.
```

Author: Modified. Thanks

# Usage

> Before modifying the Priority Class setting, all Longhorn volumes should be detached.

Reviewer: ```suggestion
Detach all Longhorn volumes before modifying Priority Class settings.
```

Author: Modified. Thanks.

> Since all Longhorn components will be restarted, the Longhorn system will temporarily be unavailable. If there are running Longhorn volumes in the system, this means only the instance manager component will be restarted eventually, and you might need to set this setting again when all Longhorn volumes are `detached`.

Reviewer: ```suggestion
Longhorn temporarily becomes unavailable when all components are restarted. However, only Instance Manager is restarted when volumes are still in use so you might be required to configure this setting again when all volumes are detached.
```

Author: Modified. Thanks.
